src/features/submission/styled.ts

```diff
@@ -0,0 +1,20 @@
+import styled from 'types/styled-components';
+
+export const SubmissionWrapper = styled.div`
+  width: 100%;
+  text-align: center;
+  padding: 16px 64px;
+
+  .ant-card {
```

Comment:
We avoid the use of css class when using styled components. Please make a new styled component with these css properties and use it instead.

E.g.
const StyledCard = styled(Card) ...